### ISSUE TITLE
Extend FECellValues to return vector valued shape functions

### ISSUE THREAD

**Issue by fredrikekre** [KristofferC/JuAFEM.jl#86]
So that for a 2Dcase:

``` jl
g = zeros(6) # residual
for gp in 1:nqp
    for i in 1:ndofs
        g[i] += (shape_symmetric_gradient(fev, qp, i) : σ) dΩ
    end
end
```

**Comment by KristofferC** [comment]
`dΩ`. Cute.

**Comment by KristofferC** [comment]
```jl
g = zeros(6) # residual
Ke = zeros(6,6) # stiffness
for gp in 1:nqp
    ɛ(i) -> shape_symmetric_gradient(fev, qp, i)
    dΩ = getdJdV(fev, qp)
    for i in 1:ndofs
        g[i] += ɛ(i) : σ * dΩ
        for j in 1:ndofs
            Ke[i,j] += ɛ(i) : C : ɛ(j) * dΩ
        end
    end
end
```

**Comment by fredrikekre** [comment]
But it should be a stiffness tensor `E` instead of compliance tensor `C`. Kappa.

**Comment by KristofferC** [comment]
Pff C is often used for stiffness: https://en.wikiversity.org/wiki/Introduction_to_Elasticity/Constitutive_relations#The_stiffness_and_compliance_tensors

**Comment by fredrikekre** [comment]
Hence the Kappa.

**Comment by KristofferC** [comment]
What kappa ;)

**Comment by fredrikekre** [comment]
`edited`
### `edited`
## `edited`
# `edited`

**Comment by fredrikekre** [comment]
Closed by #88
